images/metering-ansible-operator/roles/meteringconfig/tasks/configure_networking.yml

```diff
@@ -68,4 +68,4 @@
     _ocp_enabled_global_proxy_http_proxy: "{{ _cluster_proxy_config.status.httpProxy }}"
     _ocp_enabled_global_proxy_https_proxy: "{{ _cluster_proxy_config.status.httpsProxy }}"
   no_log: true
-  when: cluster_proxy_config is defined and cluster_proxy_config.resources | length > 0 and cluster_proxy_config.status is defined
+  when: cluster_proxy_config is defined and (cluster_proxy_config.resources | length) and _cluster_proxy_config.status is defined
```

Comment:
Looking back, this check isn't sufficient. Currently, we check whether or not this cluster-wide, proxy/cluster object exists, which is always the case when deploying Metering on OCP (unless a cluster-admin accidentally deletes this object), and whether or not this same object has a `.status` dictionary key, which also should also always be the case.

The problem, however, is when you're running a Metering installation against an OCP cluster that _hasn't_ setup a cluster-wide proxy, i.e. the proxy/cluster object will exist, but the `.status` field will be empty. We need to update the conditional to properly reflect that last point, as the current implementation only checks whether or not this dictionary key exists, and not verifying whether this key is non-empty.